app/frontend/i18n/i18n.ts

```diff
@@ -864,6 +864,9 @@ const options = {
             workType: "Work Type",
             manageButton: "Manage",
             lastUpdated: "Last updated",
+            requestNewLink: "request a new permit type",
+            requestNewPrompt:
+              "Your administrator has made the above permit types available for digital submissions. If there is another permit type you want please",
```

Comment:
If these are meant to go together, please refer to https://react.i18next.com/latest/trans-component